website/docs/r/cloudant.html.markdown

```diff
@@ -0,0 +1,131 @@
+
+* `account_id` - (String) An alpha-numeric value identifying the account ID.
+* `allow_cleanup` - (Boolean) A boolean that dictates if the resource instance should be deleted (cleaned up) during the processing of a region instance delete call.
+* `created_at` - (String) The date when the instance was created.
+* `created_by` - (String) The subject who created the instance.
+* `crn` - (String) CRN of resource instance.
+* `dashboard_url` - (String) Dashboard URL to access resource.
+* `deleted_at` - (String) The date when the instance was deleted.
+* `deleted_by` - (String) The subject who deleted the instance.
+* `extensions` - (Map) The extended metadata as a map associated with the resource instance.
+* `guid` - (String) Guid of resource instance.
+* `last_operation` - (Map) The status of the last operation requested on the instance.
+* `locked` - (Boolean) A boolean that dictates if the resource instance should be deleted (cleaned up) during the processing of a region instance delete call.
+* `plan_history` - (List of Object) The plan history of the instance.
+* `resource_aliases_url` - (String) The relative path to the resource aliases for the instance.
+* `resource_bindings_url` - (String) The relative path to the resource bindings for the instance.
+* `resource_controller_url` - (String) The URL of the IBM Cloud dashboard that can be used to explore and view details about the resource.
+* `resource_crn` - (String) The crn of the resource.
+* `resource_group_crn` - (String) The long ID (full CRN) of the resource group.
+* `resource_group_name` - (String) The resource group name in which resource is provisioned.
+* `resource_id` - (String) The unique ID of the offering.
+* `resource_keys_url` - (String) The relative path to the resource keys for the instance.
+* `resource_name` - (String) The name of the resource.
+* `resource_plan_id` - (String) The unique ID of the plan associated with the offering.
+* `resource_status` - (String) The status of the resource.
+* `restored_at` - (String) The date when the instance under reclamation was restored.
+* `restored_by` - (String) The subject who restored the instance back from reclamation.
+* `scheduled_reclaim_at` - (String) The date when the instance was scheduled for reclamation.
+* `scheduled_reclaim_by` - (String) The subject who initiated the instance reclamation.
+* `service` - (String) The service type of the instance.
+* `state` - (String) The current state of the instance.
+* `status` - (String) Status of resource instance.
+* `sub_type` - (String) The sub-type of instance, e.g. cfaas .
+* `target_crn` - (String) The full deployment CRN as defined in the global catalog.
+* `throughput` - (Map of Number) Schema for detailed information about throughput capacity with breakdown by specific throughput requests classes.
+* `type` - (String) The type of the instance, e.g. service_instance.
```

Comment:
The type of the instance, e.g. service_instance.  as

The type of the instance. For example, **service_instance**.